src/Umbraco.Core/Constants-Conventions.cs

```diff
@@ -118,25 +118,45 @@ public static class MediaTypes
                 /// </summary>
                 public const string Image = "Image";
 
+                /// <summary>
+                /// MediaType name for a video.
+                /// </summary>
+                public const string VideoName = "Video";
+
+                /// <summary>
+                /// MediaType name for an audio.
+                /// </summary>
+                public const string AudioName = "Audio";
+
+                /// <summary>
+                /// MediaType name for an article.
+                /// </summary>
+                public const string ArticleName = "Article";
+
+                /// <summary>
+                /// MediaType name for vector graphics.
+                /// </summary>
+                public const string VectorGraphicsName = "VectorGraphics";
```

Comment:
This name could be "friendlier", in the sense that it's name and not an alias, so can have spaces and punctuation.  Previously it was "Vector Graphics (SVG)".

Reply:
Ah! That's why it was hardcoded.. Fixed that up as well 👍